product-service/handlers/getProductById.js

```diff
@@ -0,0 +1,26 @@
+import products from "../DB/products";
+
+export const getProductById = async (event) => {
+  try {
+    const { id } = event.pathParameters;
+    const product = products.find((item) => item.id === parseInt(id));
```

Comment:
`https://jx4j1hwjrf.execute-api.eu-west-1.amazonaws.com/dev/products/12asdasd` returns the product with id `12`. I think that `parseInt` is redundant here.